py/bitbox02/CHANGELOG.md

```diff
@@ -2,13 +2,12 @@
 
 ## [Unreleased]
 
-# 8.0.0
-- SD card: Remove API to prompt removal of the microSD card from the device
-- Add support for regtest (supported from firmware version v9.21.0)
-
 # 7.0.0
```

Comment:
If the last released version is 6.3.0, shouldn't all these changes go under `Unreleased` for now?